lib/common_fw/common_fw.cpp

```diff
@@ -5,10 +5,15 @@
  */
 #include "common_fw.h"
 
+#define PACKET_START 0x2A
+#define PACKET_END 0xC5
```

Comment:
Are these choices arbitrary?

Reply:
Also, these are 8 bit numbers, shouldn't they be 16 bit?

Reply:
yes arbitrary. yes they should be 16.

Reply:
Ok sg. Is there any reason to choose these more carefully than just arbitrarily?